utils/enrichToken.ts

```diff
@@ -0,0 +1,47 @@
+const getCzidServicesTokenFromCookie = (cookieStr: string) => {
+  const tokenKeyVal = cookieStr
+    .split(';')
+    .map((keyValStr: string) => keyValStr.split('='))
+    .map((keyValArr: [string, string]) => ([
+      decodeURIComponent(keyValArr[0].trim()),
+      decodeURIComponent(keyValArr[1].trim()),
+    ]))
+    .find((keyValArr) => keyValArr[0] === "czid_services_token");
+
+  return tokenKeyVal ? tokenKeyVal[1] : null;
+}
```

Comment:
What should a cookieStr look like? This could be helpful context to have in a comment when reading this function so the reader can follow along with the .split() & .maps()

Reply:
These are standard HTTP cookies, so I don't think there's a need to add any specific documentation here.  I'll rename the variable to `httpCookieStr`, to be explicit about that.  The unit tests are also a good place to self-document things like this, so I'll update the unit tests to have mock cookies with multiple keys (which is closer to what they really look like).